components/RichTextEditor.js

```diff
@@ -286,6 +287,57 @@ export default class RichTextEditor extends React.Component {
     }
   }
 
+  /** A helper used by autolink to find where the url actually ends */
+  autolink_delim = url => {
+    let link_end = url.length;
+
+    while (link_end > 0) {
```

Comment:
Is that original code or is it coming from somewhere else?

If it's coming from somewhere else, could you add credits and provide the source URL as comment?

Reply:
Yes, I got the code from somewhere else. I have updated the comments to reflect this.